cerebro-template.yaml

```diff
@@ -16,13 +16,13 @@ objects:
         name: openshift-cerebro:latest
     source:
       git:
-        uri: https://github.com/jcantrill/openshift-cerebro
+        uri: https://github.com/t0ffel/openshift-cerebro
```

Comment:
Looks like we should make this a template param

Reply:
updated